<ui version="4.0" >
 <class>TextDelimiterDialog</class>
 <widget class="QDialog" name="TextDelimiterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Text Delimiter</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="3" >
       <widget class="QRadioButton" name="rbtnComma" >
        <property name="text" >
         <string>Comma ( , )</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QRadioButton" name="rbtnPipe" >
        <property name="text" >
         <string>Pipe ( | )</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QRadioButton" name="rbtnBlank" >
        <property name="text" >
         <string>Blank spaces</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QRadioButton" name="rbtnCustom" >
        <property name="text" >
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="leCustom" >
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>91</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="3" >
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Preview</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QTextEdit" name="tePreview" >
           <property name="maximumSize" >
            <size>
             <width>10000</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="cbHeader" >
           <property name="text" >
            <string>The file has a header</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextDelimiterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextDelimiterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
